db/submission_test.go

```diff
@@ -0,0 +1,58 @@
+package db
+
+import (
+    "testing"
+)
+
+func (this *DBTests) DBTestRemoveSubmission(test *testing.T) {
+    defer ResetForTesting();
+
+    testCases := []struct{targetEmail string; targetSubmission string; isRemoved bool}{
+
+        // Student, specific.
```

Comment:
In these errors, there is no longer the concept of "Student".
The DB does not care about roles, so you should clean up the comments a bit to not mention them,